src/mbgl/map/transform_state.hpp

```diff
@@ -101,7 +101,7 @@ class TransformState {
     double x = 0, y = 0;
     double angle = 0;
     double scale = 1;
-    double altitude = 1.5;
+    double FOV = 36.87;
```

Comment:
As of #3362, iOS-specific calculations assume a vertical FOV of 30°, which is consistent with MapKit.